templates/gcp/index.html

```diff
@@ -33,6 +33,16 @@ <h1>Ubuntu on Google Cloud</h1>
           <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/ybjCCu5nr6g?si=CZoIn6MtcNupEEU_" title="Secure Your Cloud Journey: Google Cloud & Ubuntu Pro for Enhanced Protection" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
         </div>
       </div>
+      <div class="p-notification--information u-no-margin--bottom">
+        <div class="p-notification__content">
+          <h5 class="p-notification__title">Discover the top open source security challenges facing 500+ organizations</h5>
+          <p class="p-notification__message">
+            <a href="https://ubuntu.com/engage/2025-state-of-software-supply-chain?utm_source=webbanner&utm_medium=organic&utm_campaign=701N100000UcKSjIAN">
```

Comment:
```suggestion
            <a href="/engage/2025-state-of-software-supply-chain?utm_source=webbanner&utm_medium=organic&utm_campaign=701N100000UcKSjIAN">
```
Can use a relative path here